### PR TITLE
update `crux_time` to not use generic response type

### PR DESCRIPTION
**Pull request by StuartHarris**
- [x] Update Time capability to not use `DateTime<Utc>` across the boundary as this doesn't work with reflection using `serde-reflect` and `serde-generate`
- [x] Improved error handling for typegen to make this more obvious if it happens again
- [x] update Rust deps
- [x] update pnpm deps

### CODE REVIEW

File: crux_time/src/lib.rs [hunk 1]
```diff
@@ -12,8 +12,11 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 pub struct TimeRequest;
 
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct TimeResponse(pub String);
+
 impl Operation for TimeRequest {
-    type Output = String;
+    type Output = TimeResponse;
```
Comment (charypar) [inline crux_time/src/lib.rs:19]: Interesting! What forced that?

File: examples/cat_facts/iOS/generated/shared.swift
Comment (charypar) [inline examples/cat_facts/iOS/generated/shared.swift:1]: Did you mean to commit these?

File: crux_time/src/lib.rs (hunk 1 above)
Comment (StuartHarris) [reply crux_time/src/lib.rs:19]: Yeah, was struggling without a wrapper. The typegen creates types we can call serialize and deserialize on

File: examples/cat_facts/iOS/generated/shared.swift
Comment (StuartHarris) [reply examples/cat_facts/iOS/generated/shared.swift:1]: we have been doing. Although I guess not strictly necessary. But if we remove them, it should be in another PR, I think

**Review by charypar — APPROVED** [review]
🚢

**Pull request merged**